src/app/copy/inline-copy/inline-copy.component.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  Component,
+  EventEmitter,
+  Input,
+  OnInit,
+  Output,
+  ViewEncapsulation
+} from '@angular/core';
+
+import { CopyService } from '../copy-service/copy.service';
+
+@Component({
+  encapsulation: ViewEncapsulation.None,
+  selector: 'pfng-inline-copy',
+  templateUrl: './inline-copy.component.html',
+  styleUrls: ['./inline-copy.component.less']
+})
+
+export class InlineCopyComponent implements OnInit {
+  @Input('label') label: string;
+  @Input('token') token: string = 'Missing \'token\' @Input property';
+  @Input('copyBtnTxt') copyBtnTxt: string = 'Copy';
+
+  @Output('copiedToClipboard') copiedToClipboard = new EventEmitter();
+
+  /**
+   * The default constructor
+   */
+  constructor(
+    private copyService: CopyService) {}
+
+  ngOnInit(): void {
+    if (!this.label) throw new Error('Missing required @Input property \'label\'');
```

Comment:
I don't believe we should force a tooltip or aria-label.

Reply:
I see, in an effort to keep the component as flexible as possible, I'll change this so it doesn't throw an error.

One benefit of making this required was that I was then able to construct sensible defaults for the aria labels, in case users didn't add them theirselves. The problem that came up was when there were several being used on a same page, from an accessibility perspective you couldn't tell them apart. "Copy Button, Copy Button, Copy Button" for example. So, it was kinda nice being able to ensure this aspect of accessibility wasn't neglected.

But yea, I'll remove this check from the core component and just try to hit that point home with one of the examples.